CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ FetchContent_Declare(
   bm_common
   GIT_REPOSITORY https://github.com/bristlemouth/bm_common.git
   #TODO: This must be set by configuration somehow
-  GIT_TAG        main
+  GIT_TAG        victor/adding_more_abstractions
```

Comment:
TODO - reference back to main once https://github.com/bristlemouth/bm_common/pull/2 gets merged

Reply:
Done!